### PR TITLE
bump go to 1.21

### PR DESCRIPTION
**Pull request by yxxhero**

### CODE REVIEW

File: pkg/app/app.go [hunk 1]
```diff
@@ -582,7 +582,7 @@ func (a *App) PrintState(c StateConfigProvider) error {
 			errs = append(errs, err)
 		}
 
-		return
```
Comment (mumoshu) [inline pkg/app/app.go:585]: Ohh is this unallowed in Go 1.21 now @yxxhero ?

Comment (yxxhero) [reply pkg/app/app.go:585]: no. because the nakedret, linter of golint-ci. shall we disable the nakedret linter? @mumoshu WDYT?

Comment (mumoshu) [reply pkg/app/app.go]: Awesome @yxxhero! I'm new to that code review guideline and the nakedret linter. Makes sense. And keep this for now and revising the appropriate setting for nakedret sounds good to me!

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks for your contribution @yxxhero ☺️

**Pull request merged**